server/routes/functionalSkillsResolver.test.ts

```diff
@@ -1,6 +1,6 @@
 import moment from 'moment/moment'
```

Comment:
The file `mostRecentFunctionalSkillsResolver.ts` and its test `mostRecentFunctionalSkillsResolver.test.ts` have been moved out of the `routes/overview` folder, up one level to `routes`; and renamed to be simply `functionalSkillsResolver`

This is because we now have 2 routes and controllers in different folders wanting to use this module, so it made sense to move it up a level. Plus in my next PR I will be adding and exporting a new function from it - `allFunctionalSkillsResolver` - so in readiness for that I have renamed the module to simply `functionalSkillsResolver` so it will work for both use cases